website/thaliawebsite/settings/settings.py

```diff
@@ -310,7 +310,7 @@
 )
 
 # Payment settings
-THALIA_PAY_ENABLED_PAYMENT_METHOD = False
+THALIA_PAY_ENABLED_PAYMENT_METHOD = True
```

Comment:
Don't forget to change this to `False`. If you want to make sure this does not happen you should create a `localsettings.py` in `thaliawebsite/settings` with `THALIA_PAY_ENABLED_PAYMENT_METHOD = True`. That will override it locally and will not be committed since that file is a part of the gitignore.